examples_server/examples/data_table/basic/index.html

```diff
@@ -27,7 +27,6 @@ <h2>Basic Table</h2>
     <div id="example-area"></div>
 
     <!-- libraries -->
-    <script type="text/javascript" src="../../../vendor/json2.js"></script>
```

Comment:
so this PR shares a lot of changes that I also saw in https://github.com/Invoca/backdraft/pull/101 - is the plan to get 101 into master and rebase this one?  I'd be in favor of that (and I'm good if 101 goes to master now)

Reply:
#101 is now merged to master. This branch was daisy-changed off #101's branch.